src/django/api/management/commands/backfill_moderation_event_os_id.py

```
@@ -0,0 +1,79 @@
+from django.core.management.base import BaseCommand
+from opensearchpy.exceptions import ConnectionError, NotFoundError
```

Ruff (0.15.12)

[error] Import `ConnectionError` is shadowing a Python builtin

(A004)